packages/server/utils/analytics/analytics.ts

```diff
@@ -387,6 +388,20 @@ class Analytics {
       newName
     })
 
+  billingLeaderModified = (
+    userId: string,
```

Comment:
In `analytics.ts`, `userId` is usually an argument, but we're passing in the `viewerId`, e.g. [here](https://github.com/ParabolInc/parabol/blob/4d892690c8f5a252e884a80b49ff3f31be9bbe2f/packages/server/graphql/mutations/startCheckIn.ts#L101) and [here](https://github.com/ParabolInc/parabol/blob/e17d1124d06c553751f6262be2d1ad0477b034c2/packages/server/graphql/mutations/removeAtlassianAuth.ts). 

When adding/removing the billing leader, the viewer can be different from the user that is being updated, and we probably want to track both.